test/Microsoft.ML.Tests/Scenarios/OvaTest.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Trainers.FastTree;
+using Microsoft.ML.Trainers.Online;
+using Xunit;
+
+namespace Microsoft.ML.Scenarios
+{
+    public partial class ScenariosTests
+    {
+        [Fact]
+        public void OvaLogisticRegression()
```

Comment:
Do you want to bring over the TLC tests with baselines?

Reply:
Compared to internal repo, the baseline numbers still differ. I think there are more subtle bugs with OVA . Once we figure out root cause of those, we can have a separate PR with maml based command line tests .  

This PR addresses the LinearSVM issue, and tests for it. 

---
In reply to: [233720493](https://github.com/dotnet/machinelearning/pull/1574#discussion_r233720493) [](ancestors = 233720493)

Reply:
Can you please remove the link to issue 1387 then? I am not sure whether the issue will be automatically closed when this PR gets merged, but if it doesn't solve the issue it should not be closed.

---
In reply to: [233744115](https://github.com/dotnet/machinelearning/pull/1574#discussion_r233744115) [](ancestors = 233744115,233720493)

Reply:
removed link so 1387 is not auto closed.

---
In reply to: [234282986](https://github.com/dotnet/machinelearning/pull/1574#discussion_r234282986) [](ancestors = 234282986,233744115,233720493)